vaxrank/cli.py

```diff
@@ -120,13 +133,17 @@ def main(args_list=None):
 
     df.to_csv(args.output_csv, index=False)
 
-    ascii_report = ascii_report_from_ranked_vaccine_peptides(
+    make_ascii_report_from_ranked_vaccine_peptides(
         ranked_variants_with_vaccine_peptides=ranked_list,
         mhc_alleles=mhc_alleles,
         variants=variants,
-        bam_path=args.bam)
+        bam_path=args.bam,
+        ascii_report_path=args.output_ascii_report)
 
-    print(ascii_report)
-
-    with open(args.output_ascii_report, "w") as f:
-        f.write(ascii_report)
+    make_html_report_from_ranked_vaccine_peptides(
```

Comment:
👍  for optional reports. Ultimately our reports will have information coming from sources other than vaxrank so we'll want some way to grab structured output from vaxrank and not the report itself.